core/runtime/common/runtime_environment_factory.cpp

```diff
@@ -99,7 +99,7 @@ namespace kagome::runtime {
     } else {
       if (auto res = env.storage_provider->setToEphemeralAt(storage_state_);
           !res) {
-        parent_factory->logger_->error(
+        parent_factory->logger_->debug(
```

Comment:
please keep it as error or change it to warning. I will fix the cause of these excessive logs in one of next PRs